taxi/models.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ("name", )
+
+    def __str__(self):
+        return self.name
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer,
+                                     on_delete=models.CASCADE,
+                                     related_name="cars")
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name="cars")
```

Comment:
it's recommended to start parameters from the next line instead of spliting them in the way you did